apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.spec.tsx

```diff
@@ -0,0 +1,110 @@
+import { MockedProvider, MockedResponse } from '@apollo/client/testing'
+import { render } from '@testing-library/react'
+
+import { FlagsProvider } from '@core/shared/ui/FlagsProvider'
+
+import {
+  ThemeMode,
+  ThemeName
+} from '../../../../../../../__generated__/globalTypes'
+import {
+  StepAndCardBlockCreate,
+  StepAndCardBlockCreateVariables
+} from '../../../../../../../__generated__/StepAndCardBlockCreate'
+import { STEP_AND_CARD_BLOCK_CREATE } from '../../../../../../libs/useStepAndCardBlockCreateMutation/useStepAndCardBlockCreateMutation'
+import { ThemeProvider } from '../../../../../ThemeProvider'
+
+import { AddBlock } from './AddBlock'
```

Comment:
We do this to also test the `index.ts` file of this folder
```suggestion
import { AddBlock } from '.'
```